pkg/client/interfaces.go

```diff
@@ -290,6 +305,20 @@ func (o *ListOptions) InNamespace(ns string) *ListOptions {
 	return o
 }
 
+// SetLimit is a convenience function that sets the limit, and
+// then returns the options. It mutates the list options.
+func (o *ListOptions) SetLimit(n int64) *ListOptions {
+	o.Limit = n
+	return o
+}
+
+// SetContinueToken is a convenience function that sets the continuation
+// token, and then returns the options. It mutates the list options.
+func (o *ListOptions) SetContinueToken(token string) *ListOptions {
```

Comment:
should probably match the name of the corresponding `ListOptionFunc` (i.e. the name should be `SetContinue`, or `Continue` should be `ContinueToken`)